src/server/server.ts

```diff
@@ -51,11 +50,8 @@ export class AuthlessServer {
     }
   }
 
-  private async getJsonResponse (page: Page, bot?: IBot | AnonBot): Promise<string> {
-    let username = 'anonymous'
-    if(bot instanceof Bot) {
-      username = bot.username
-    }
+  private async getJsonResponse (page: Page, bot?: IBot): Promise<string> {
+    let username = bot?.username ?? 'anonymous'
     return JSON.stringify({
       meta: {
         url: page.url(),
```

Comment:
I must have missed that before - why is it returning 
```suggestion
    return JSON.stringify({
      meta: {
        url: page.url(),
        username
      },
      content: await page.content(),
      xhrs: this.responses
```

`content` is deprecated and where is `page` and `main` attributes?